ST_mortality.c

```diff
@@ -692,31 +776,37 @@ static void _age_independent( const SppIndex sp) {
 
 }
 
-/***********************************************************/
+/**
+ * \brief Kills individuals to simulate death by insufficient resources.
+ * 
+ * Note that this function does NOT check if resource availibility is low.
+ * It kills an amount of individuals proportional to resource limitations,
+ * but deciding if any individuals should be killed at all (whether or
+ * not to call this function) takes place in mort_Main().
+ * 
+ * In reality, resource death would occur gradually over the growing
+ * season. However, in this yearly-time-step model, it must be done either
+ * before or after the growing season. 
+ * 
+ * This function takes an additional step with clonal species. It calls
+ * _stretched_clonal() to kill additional amounts of clonal plants due 
+ * to insufficient resources.
+ * 
+ * Initial programming by Chris Bennett @ LTER-CSU 6/15/2000.
+ * 
+ * \param rg The Index in RGroup of the group to kill.
+ * 
+ * \sideeffect If this function decided to kill any individuals, which
+ *             it most likely will, the individual linked lists of every
+ *             species in RGroup[rg] will be modified.\n
+ *             _SomeKillage set to TRUE if an individual was killed.
+ * 
+ * \sa _stretched_clonal() which is called to perfom additional mortality.
+ * \sa mort_Main() where _no_resources() is called.
+ * 
+ * \ingroup MORTALITY_PRIVATE
+ */
 static void _no_resources( GrpIndex rg) {
-/*======================================================*/
-/* use EQN 7, 8, 9 prior to growing
- * Resource limitation results in plant mortality
- * (ie, individuals or portions of individuals). In reality, this
- * would happen gradually over the season, but in this
- * yearly-time-step model it has to be done either before
- * or after the growth routine.  C&L 1990 (p241) note that
- * growth rates are also reduced--this is done in the main
- * growth loop by setting gmod = 1/PR.
-
- * Note to make it here, the rgroup's PR MUST BE > 1.0., which is checked in mort_Main().
```

Comment:
I would like to retain the above commend regarding PR.

Reply:
Addressed by commit 984bd13.